tests/kit/unique.py

```diff
@@ -0,0 +1,22 @@
+"""Test unique."""
+
+
+# --- import -------------------------------------------------------------------------------------
+
+
+import numpy as np
+
+import WrightTools as wt
+
+
+# --- test ---------------------------------------------------------------------------------------
+
+
+def test_5():
+    arr = np.array([0, 1, 2, 3, 3 + 1e-7])
+    assert wt.kit.unique(arr).all() == np.array([0, 1, 2, 3.000005]).all()
```

Comment:
Floating point arithmetic can be tricky here... be careful about using '==' this may not work on all machines